### PR TITLE
Update ens wrapper to single module and renaming changes

### PR DESCRIPTION
**Pull request by nerfZael**
Recipes are not upgraded. I'll create an issue to migrate them to workflows

### CODE REVIEW

File: protocol/ethereum/ens/wrapper/package.json [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-    "build": "npx w3 build",
-    "test:env:up": "npx w3 test-env up",
-    "test:env:down": "npx w3 test-env down",
-    "deploy": "yarn deploy:contract && yarn deploy:web3api",
-    "deploy:web3api": "npx w3 build --ipfs http://localhost:5001 --test-ens ens.eth",
+    "build": "npx polywrap build",
+    "test:env:up": "npx polywrap test-env up",
+    "test:env:down": "npx polywrap test-env down",
+    "deploy": "yarn deploy:contract && yarn deploy:polywrap",
+    "deploy:polywrap": "npx polywrap build --ipfs http://localhost:5001 --test-ens ens.eth",
```
Comment (cbrzn) [inline protocol/ethereum/ens/wrapper/package.json:11]: Let's update this and use the deploy manifest - You can check it out here

File: protocol/ethereum/ens/wrapper/package.json [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
     "deploy:contract": "node ./scripts/deploy-contracts.js",
-    "test": "yarn test:e2e && yarn test:env:up && yarn deploy && yarn test:recipe && yarn test:env:down",
+    "test": "yarn test:e2e && yarn test:env:up && yarn deploy && yarn test:env:down",
     "test:e2e": "jest --passWithNoTests --runInBand --verbose",
-    "test:recipe": "npx w3 query ./recipes/e2e.json"
+    "test:recipe": "npx polywrap query ./recipes/e2e.json"
```
Comment (cbrzn) [inline protocol/ethereum/ens/wrapper/package.json:15]: Let's remove this for now

File: protocol/ethereum/ens/wrapper/polywrap.build.yaml [hunk 3]
```diff
@@ -5,8 +5,7 @@ config:
   node_version: "14.16.0"
   include:
     - ./package.json
-    - src/query
-    - src/mutation
+    - src/index.ts
     - src/common
```
Comment (cbrzn) [inline protocol/ethereum/ens/wrapper/polywrap.build.yaml:9]: I think we should not keep the folder common and move everything to one place

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]
LGTM, just wrapper needs rename IMO.

File: protocol/ethereum/ens/wrapper/package.json [hunk 4]
```diff
@@ -1,27 +1,26 @@
 {
-  "name": "@web3api/ens-api",
+  "name": "@polywrap/ens-api",
```
[the remark this answers is not in the document]
Comment (cbrzn) [reply protocol/ethereum/ens/wrapper/package.json:2]: uuu nice catch

**Review by cbrzn — APPROVED** [review]
Once Niraj comment has been addressed this is good to go :green_apple:

**Review by Niraj-Kamdar — APPROVED** [review]
LGTM!

**Pull request merged**